src/libs/TripReservationUtils.ts

```diff
@@ -24,4 +25,26 @@ function getReservationsFromTripTransactions(transactions: Transaction[]): Reser
         .flat();
 }
 
-export {getTripReservationIcon, getReservationsFromTripTransactions};
+type TripEReceiptData = {
+    /** Icon asset associated with the type of trip reservation */
+    tripIcon?: IconAsset;
+
+    /** EReceipt background color associated with the type of trip reservation */
+    tripBGColor?: EReceiptColorName;
+};
+
+function getTripEReceiptData(transaction?: Transaction): TripEReceiptData {
+    const reservationType = transaction ? transaction.receipt?.reservationList?.[0]?.type : '';
+
+    switch (reservationType) {
+        case CONST.RESERVATION_TYPE.FLIGHT:
+        case CONST.RESERVATION_TYPE.CAR:
+            return {tripIcon: Expensicons.Plane, tripBGColor: CONST.ERECEIPT_COLORS.PINK};
```

Comment:
We need to remove tripBGColor and calculate the color combos using the same methods in https://github.com/Expensify/App/blob/main/src/components/EReceipt.tsx#L35-L36